src/gui/search/searchtab.h

```diff
@@ -32,49 +32,73 @@
 #define SEARCHTAB_H
 
 #include <QWidget>
+#include "searchsortmodel.h"
+
+#define ENGINE_URL_COLUMN 4
+#define URL_COLUMN 5
 
 class QLabel;
 class QTreeView;
 class QHeaderView;
 class QStandardItemModel;
-class QSortFilterProxyModel;
-class QModelIndex;
 class QVBoxLayout;
 
 class SearchSortModel;
 class SearchListDelegate;
 class SearchWidget;
 
-class SearchTab: public QWidget
+namespace Ui
 {
+    class SearchTab;
+}
+
+class SearchTab: public QWidget {
```

Comment:
`{` to the next line
We use `{` in the same line only after control statements (if, else, for, while, switch etc.). See CODING_GUIDELINES.md.